CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+# [7.19.0](https://github.com/dashevo/wallet-lib/compare/v7.18.1...v7.19.0) (2021-05-03)
+
+
+### Features
+
+* update dpp with verifyInstantLock method ([#234](https://github.com/dashevo/wallet-lib/issues/234))
+
+
+
```

Comment:
BREAKING CHANGES:
- Account#fetchStatus response format is changed and not compatible with the previous version
- Transport#getStatus responded format is changed and not compatible with the previous version